src/components/file-upload/index.md.njk

```diff
@@ -23,6 +23,54 @@ There are 2 ways to use the file upload component. You can use HTML or, if you
 
 {{ example({group: "components", item: "file-upload", example: "default", html: true, nunjucks: true, open: false, id: "default-2"}) }}
 
+### Error messages
+
+Check that a valid date is entered by the user. Invalid dates should be reported to the user like this:
```

Comment:
These error messages don't relate to dates.

Reply:
Good spot. I have changed this to match other documentation in the design system.